src/inthandler.S

```diff
@@ -11,134 +11,163 @@
 inthandler:
 	.global inthandler
 
-	/*Save $2 before using*/
-	sd $2,save02
-
 	.set noat
-	/*Fetch exception pc off cop#0*/
-	mfc0 $2,C0_EPC
-
-	la k1,save01
-	sd $1,(k1)
+	.set noreorder
```

Comment:
Without a matching `.set reorder`, we may be disabling them in unexpected ways.

Reply:
Isn't this per-file?

Reply:
I'm not 100% sure on that. I suspect they might get appended to each other in the end.

Reply:
Each assembly file generates its own object file. Disassembling inthandler.o I don't see any special symbols being emitted related to the state of `.set noreorder` so I would say that this is local state of the assembler that is not persisted across different files.